### ISSUE TITLE
Storing manual flag for loggers data point in odm2.

### ISSUE THREAD

**Issue by ZofiaRudjord** [NIVANorge/odm2-postgres-api#167]
Logger data are stored in odm2 in TimeSeriesResultValues table
http://odm2.github.io/ODM2/schemas/ODM2_Current/tables/ODM2Results_TimeSeriesResultValues.html

Data Quality flag is stored in the QualityCodeCV column, and takes values : good, bad  or unknown.
These flags are derived using automated quality checks (typically range test, frozen test and spike test).
Those tests are registered in the odm2. For each time series, there is a link between time series and the quality tests,
indicating which tests ran.

It has been requested to import historical data from nivabasen. Those data were manually controlled
by data owners and a flag was assigned and stored in nivabasen.

There are 2 options for storing this manual flag:
1.  In QualityCodeCV column as for automated flag. This will indicate that this flag was a result of the automated tests which is wrong.
2. In CensorCodeCV column. This will indicate that data were censored, which they were in a way. In order to do so I would have to extend cv_sensorcode with: Approved, NotApproved.

Unless there are objections @jemmima @haakondr  I will go for option 2.

**Comment by ZofiaRudjord** [comment]
This is now implemented

**Comment by haakondr** [comment]
Sounds good to me.

Just thinking out loud here:

is it impractical to operate with two columns (QualityCodeCV **and** CensorCodeCV) indicating the validity of an TimeSeriesResultValues? 

For option 1, could it be an option to add a custom vocabulary **InvalidManualFlagged** or something like that?

I am fine with either approach though, but I think we should differentiate manual and automatic QC, that's for sure.

**Comment by haakondr** [comment]
another thought, would it make sense to store who flagged the value, and why?

**Comment by ZofiaRudjord** [comment]
I agree it is impractical to have two columns. Let's extend the CVs.

**Comment by ZofiaRudjord** [comment]
Yes, the info who flagged it and why is required at ingestion if ingesting with manual flags. 
It is required in the data file and stored in the processed-logger-files bucket.

To store who flags we could 1) add annotation (its possible per point but cumbersom) 2) modify the table.
I would rather go for opt 2) if we want it in the database.

For historical data this info is in nivabase (for most of the cases).
